src/App.tsx

```diff
@@ -1,24 +1,48 @@
-import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import './SectionView.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFaceSmile } from '@fortawesome/free-solid-svg-icons';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="parent">
+      <div className="section-title">
+        <div className='text'>
+          <h1> Intro </h1>
+          <p>This intro section consists of a tuba quartet in the style of DJ Templeton & The Windsurfers</p>
+        </div>
+        <div className='buttons'>
+          <button> 42 revisions </button>
+          <button> save revision </button>
+        </div>
+      </div>
+      <div className="chords">
+        <ol>
+          <li>C</li>
+          <li>Dm</li>
+          <li>F</li>
+          <li>G</li>
+        </ol>
+      </div>
+      <div className="files">
+        <span>+ Files</span>
+          <div>Bass.mp3 <br></br> <br></br> 2 Comments</div>
+          <div>Drums.mp3 <br></br> <br></br> 2 Comments</div>
+          <div>Yodeling.mp3 <br></br> <br></br> 2 Comments  </div>
+          <div>Tuba.mp3 <br></br> <br></br> 2 Comments  </div>
```

Comment:
Example of how to use `map` make a list of React component children from the original array of data:

```ts
const filesData = [
    {
        id: '1',
        name: 'Bass.mp3',
        numComments: 2,
    }
    // ...
];
```

```suggestion
          {filesData.map((fileData) => (
              <div key={fileData.id}>
                  {fileData.name}
                  <br/><br/>
                  {fileData.numComments} Comments
              </div>
          ))}
```

Or

```ts
          {filesData.map((fileData) => (
              <SectionAttachedFile fileData={fileData}/>
          ))}
```

We'll see if it makes sense to make it its own component. I trust your judgment at the current state.